tests/network/localnet/liblocalnet.py

```diff
@@ -146,3 +163,43 @@ def client_server_active_connection(
             server_port=port,
         ) as client:
             yield client, server
+
+
+def create_nncp_localnet_on_secondary_node_nic(
+    worker_node: Node, nodes_available_nics: dict[str, list[str]], mtu: int | None = None
+) -> Generator[libnncp.NodeNetworkConfigurationPolicy, None, None]:
+    bridge_name = "localnet-ovs-br"
+    desired_state = libnncp.DesiredState(
+        interfaces=[
+            libnncp.Interface(
+                name=bridge_name,
+                type=NNCP_INTERFACE_TYPE_OVS_BRIDGE,
+                mtu=mtu,
+                ipv4=libnncp.IPv4(enabled=False),
+                ipv6=libnncp.IPv6(enabled=False),
+                state=libnncp.Resource.Interface.State.UP,
+                bridge=libnncp.Bridge(
+                    options=libnncp.BridgeOptions(libnncp.STP(enabled=False)),
+                    port=[
+                        libnncp.Port(
+                            name=nodes_available_nics[worker_node.name][-1],
+                        )
+                    ],
+                ),
+            )
+        ],
+        ovn=libnncp.OVN([
+            libnncp.BridgeMappings(
+                localnet=LOCALNET_OVS_BRIDGE_NETWORK,
+                bridge=bridge_name,
+                state=libnncp.BridgeMappings.State.PRESENT.value,
+            )
+        ]),
+    )
+    with libnncp.NodeNetworkConfigurationPolicy(
```

Comment:
@EdDev The team should prioritize adding `client` in the existing code to minimize changes in newly added code.

Reply:
Yes, that is what @rnetser meant. The new rules require an explicit specification of the client and prioritizing it be be unpriv when possible.
You can take it as a follow up.